src/server.js

```diff
@@ -23,6 +24,10 @@ const { Version } = require('./utility/version');
 // Raise AssertionError if environment variables are not set
 validateEnvironment(ENVIRONMENT);
 
+if (ENVIRONMENT.DATABASE_USE_SSL) {
+    validateEnvironment([ENVIRONMENT_DATABASE_SSL]);
+}
+
```

Comment:
One other problem, `validateEnvironment` for `ENVIRONMENT_DATABASE_SSL` never has a chance to run in the first place when `DATABASE_USE_SSL=1` and `SSL_CA` is blank. This is because `readSSLCAFile` executes first, and throws:

```
ERROR: Failed to read SSL CA file. SSL is enabled via the DATABASE_USE_SSL environment variable.
Check the path defined in .env as SSL_CA.:
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL.
Received undefined
```

Reply:
Indeed, sorry about that. I found out that when the SQL query runner is loaded (indirectly via importing some of the modules which import it) it instantiates the connection pool right away which causes this issue. I moved those imports below the validation.